city_scrapers/spiders/il_sports_facilities_authority.py

```diff
@@ -0,0 +1,104 @@
+from datetime import datetime, timedelta
+
+from city_scrapers_core.constants import BOARD
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+
+
+class IlSportsFacilitiesAuthoritySpider(CityScrapersSpider):
+    name = "il_sports_facilities_authority"
+    agency = "Illinois Sports Facilities Authority"
+    timezone = "America/Chicago"
+    allowed_domains = ["www.isfauthority.com"]
+    start_urls = ["https://www.isfauthority.com/governance/board-meetings/"]
+    location = {
+        'name': 'Authority offices',
+        'address': 'Guaranteed Rate Field, 333 West 35th Street, Chicago, IL'
+    }
+
+    def parse(self, response):
+        """
+        `parse` should always `yield` Meeting items.
+
+        Change the `_parse_title`, `_parse_start`, etc methods to fit your scraping
+        needs.
+        """
+        for item in response.css('div.wpb_text_column div.wpb_wrapper div.inner-text h2'):
```

Comment:
In general, it looks like most of the logic between the H2 and general items is the same, so if possible it would be good to consolidate these loops

Reply:
you mean add items to one list from both responses and then parse?

Reply:
Or even just modify the loop selector to:

```python
for item in response.css('div.wpb_text_column div.wpb_wrapper div.inner-text h2, div.wpb_text_column div.wpb_wrapper p'):
    meeting = Meeting(...
```

Reply:
done :)